autoload/ale/completion.vim

```diff
@@ -262,6 +262,24 @@ function! ale#completion#Show(result) abort
         \   {-> ale#util#FeedKeys("\<Plug>(ale_show_completion_menu)")}
         \)
     endif
+
+    if l:source is# 'ale-callback'
+        call ale#completion#CallbackComplete()
+    endif
+endfunction
+
+function! ale#completion#CallbackComplete() abort
+    call b:asyncomplete_callback(b:ale_completion_result)
+endfunction
```

Comment:
Remove this function, and put the call for the callback inline in the function above.